Updated Korean Dynamic City Names

mechu1205 opened the pull request:
-Pusan changed to Busan
-Busan is named Dongrae when Korea settles it
-Dongrae changes name to Busan when someone else owns it or Korea enters the Ind Age
-Updated Korean city names in Korea, Manchuria, China and Japan.
-This PR includes the GP names list update for Korea.

mechu1205 committed:
Add files via upload

Updated Korean GP names.

mechu1205 committed:
Updated Korean GP names

mechu1205 committed:
korean city name update; bug test

mechu1205 committed:
Reverted CityNameManager

Now this branch has only the GP names change in effect

mechu1205 committed:
Updated Korean City Names

Changes
-'Pusan' has been changed to 'Busan'
-Busan is named 'Dongrae' when Korea settles it.
-Dongrae changes name to Busan when someone else owns it or Korea enters the Industrial Era.
-Updated Korean city names in Manchuria, China, and Japan
-Korean city names in Manchuria are mostly based on city names from the Balhae Kingdom.

dguenms commented on Assets/Python/CityNameManager.py:
```diff
@@ -276,6 +277,7 @@ def revertCommunistNames(iCiv):
 	'Ayutthaya'		:	'Bangkok',
 	'York'			:	'Toronto',
 	'Barara'		:	'Addis Ababa',
+	'Dongrae'		:	'Busan',
```
What is the significance of Dongrae?

mechu1205 replied:
Dongrae was the name of the larger district, and Busan was part of it.
After Japan colonized Korea they named the entire area as the city of Busan,
and Dongrae became a part of the city.
This is still the case after the independence of Korea,
so with this update Dongrae will change name to Busan in-game if
a) another nation acquires it from Korea
b) Korea enters the Industrial Era

dguenms commented on Assets/Python/CityNameManager.py:
```diff
@@ -3372,8 +3377,11 @@ def onRevolution(iCiv):
 	"Budin"			:	"Budapest",
 	"Ofen"			:	"Budapest",
-	"Buju"			:	"Buju",
-	"Tieling"		:	"Buju",
```
What happened to this?

mechu1205 replied:
Tieling was an unused city name.
Buju was a Korean city name for a tile in Manchuria.
That was wrongly placed so I fixed it.
Buju is now indexed to Fushun.

dguenms replied:
So no city is called Buju anymore?

mechu1205 replied:
No, it is now a name for a different tile which is indexed as Fushun.

dguenms commented on Assets/Python/CityNameManager.py:
```diff
@@ -3463,6 +3475,7 @@ def onRevolution(iCiv):
 	"Chngching"		:	"Chongqing",
 	"Chongqing"		:	"Chongqing",
 	"Jousai"		:	"Chongqing",
+	"Junggyeong"	:	"Chongqing",
```
There is a tab missing here.

mechu1205 replied:
Oops

dguenms commented on Assets/Python/CityNameManager.py:
```diff
@@ -3857,12 +3893,14 @@ def onRevolution(iCiv):
 	"Kaesong"		:	"Kaesong",
 	"Kaijo"			:	"Kaesong",
-	"Songak"		:	"Kaesong",
+	"Kaesong"		:	"Kaesong",
```
What happened to Songak? `Kaesong -> Kaesong` twice seems like a mistake.

mechu1205 replied:
Removed one of the duplicated lines.
The proper name is Kaesong,
Songak was more like a nickname and was used officially for only a short time.

dguenms commented on Assets/Python/CityNameManager.py:
```diff
@@ -4202,25 +4253,33 @@ def onRevolution(iCiv):
+	"Janggi"		:	"Nagasaki",
 	"Mingguwu"		:	"Nagoya",
 	"Nagoya"		:	"Nagoya",
+	"Myeonggo'ok"	:	"Nagoya",
```
There is a tab missing here.

mechu1205 replied:
Oops

dguenms commented on Assets/Python/CityNameManager.py:
```diff
@@ -4864,6 +4934,7 @@ def onRevolution(iCiv):
 	"Jianghu"		:	"Toukyou",
 	"Tokio"			:	"Toukyou",
 	"Toukyou"		:	"Toukyou",
+	"Donggyeong"	:	"Toukyou",
```
There is a tab missing here.

dguenms commented on Assets/Python/CityNameManager.py:
```diff
@@ -4974,6 +5046,10 @@ def onRevolution(iCiv):
 	"Vladivostok"		:	"Vladivostok",
+	"Haesamwi"		:	"Vladivostok",
+	"Hwaju"			:	"Vladivostok",
+	"Anju"			:	"Vladivostok",
```
What are all these cities? They can't all be Vladivostok.

mechu1205 replied:
There are 3 tiles marked as Vladivostok in the Russian map.
Hwaju, Anju and Jeongju are names of cities that existed in that area.
After the fall of the Balhae Kingdom that area was lost to Korea.
After Vladivostok was settled, Koreans called it Haesamwi.

In-game, if Korea settles that area the city will be named as Hwaju/Anju/Jeongju.
If they take it from another civ it will be renamed as Haesamwi.

dguenms commented on Assets/Python/CityNameManager.py:
```diff
@@ -5626,14 +5706,84 @@ def onRevolution(iCiv):
 {
 	"Beijing"		:	"Bukgyeong",
-	"Zhongdu"		:	"Bukgyeong",
+	"Zhongdu"		:	"Bukpyeong",
```
This looks like a typo.

mechu1205 replied:
Busan? No, that's the official English translation of the city's name.

dguenms commented on Assets/Python/CityNameManager.py:
```diff
@@ -5626,14 +5706,84 @@ def onRevolution(iCiv):
+	"Busan"			:	"Busan",
 	"Pyongyang"		:	"Pyongyang",
-	"Shenyang"		:	"Seonyang",
+	"Shenyang"		:	"Shimyang",
```
That's not how I usually see that transliterated.

mechu1205 replied:
There are two ways to read the Chinese letter 'Shen'.
The appropriate way in Korea is 'Shim'.
A minor fix.

mbird1205 replied:
Guangzhou - Guangju is to avoid confusion with Gwangju in the Korean penn.
Bukpyeong is the translation for Beiping, one of the ancient/medieval names of Beijing.

dguenms commented on Assets/Python/CityNameManager.py:
```diff
@@ -5626,14 +5706,84 @@ def onRevolution(iCiv):
+	"Anyang"		:	"Anyang",
+	"Anshan"		:	"Ansan",
+	"Aomori"		:	"Cheongsam",
+	"Mudanjiang"	:	"Mokdangang",
```
Tab.

mechu1205 replied:
Oops

dguenms commented on Assets/Python/CityNameManager.py:
```diff
@@ -5626,14 +5706,84 @@ def onRevolution(iCiv):
+	"Wuhan"			:	"Muhan",
+	"Qiqihaer"		:	"Chichihar",
+	"Haerbin"		:	"Harbin",
+	"Zhangjiakou"	:	"Janggagu",
```
Tab.

dguenms commented on Assets/Python/CityNameManager.py:
```diff
@@ -5626,14 +5706,84 @@ def onRevolution(iCiv):
+	"Chongqing"		:	"Junggyeong",
+	"Kagoshima"		:	"Nokado",
+	"Kanazawa"		:	"Geumtaek",
+	"Khabarovsk"	:	"Baekryeok",
```
Tab.

dguenms commented on Assets/Python/CityNameManager.py:
```diff
@@ -5626,14 +5706,84 @@ def onRevolution(iCiv):
+	"Gaoxiong"		:	"Jukrim",
+	"Toukyou"		:	"Donggyeong",
+	"Toyohara"		:	"Pungwon",
+	"Vladivostok"	:	"Haesamwi",
```
Tab.

dguenms reviewed (changes requested):
Can you exclude the GP changes here so it is easier to merge independently?

To be honest, I would also appreciate if you could separate the addition of entirely new city names from simple translations of existing Japanese and Chinese names.

mechu1205 committed:
Reverted GP names list, Minor fix on CityNameManager

The CityNameManager had a problem of Korean Euiju/Shinju not changing name when captured by China.
This is a problem that existed before my update, so in the main branch Korean Sariwon won't change name to Sanshan when China captures it.
This is because Sanshan wasn't indexed in Chinese city names list.
I fixed this alongside with Fushun (1~2 tiles north to Sanshan).
Now when Korea settles Euiju/Shinju and China captures it, it will change name to Sanshan.
When Korea captures the city from China the name will change to Daeryeon
(the Korean pronunciation for the Chinese city name)
This is intended because IRL that territory is controlled by China.

mechu1205 committed:
Correct number of tabs

tabtab

mechu1205 commented:
'I would also appreciate if you could separate the addition of entirely new city names from simple translations of existing Japanese and Chinese names.'
How do you mean?
Like create separate commits?

dguenms commented:
I mean separate pull requests.

mechu1205 commented:
Oh I see.

mechu1205 committed:
Corrected translation for Gaoxiong

mechu1205 committed:
Corrected translation for Gaoxiong

oops wrong file. fixed that.

mechu1205 committed:
deleted file in wrong location

mechu1205 committed:
Correct location of file

mechu1205 committed:
Original GP names file

for some reason github thought i made a change

mechu1205 committed:
deleted file in wrong location

dammit

mechu1205 committed:
correct number of tabs

mechu1205 committed:
Changed Sanshan to Dalian

Dalian is much bigger and much more important than Sanshan.
Also all other civs - Jap, Tib, Mon, etc refers the area to some form of the word 'Dalian'.
So I changed the Chinese name as well for consistency.

mechu1205 commented:
Created a separate PR with minimal changes.
